### ISSUE TITLE
renault.charge_schedule service not reflecting the "activated" attribute

### ISSUE THREAD

**Issue by eviollet** [hacf-fr/hassRenaultZE#110]
The "activated" attribute isn't reflected correctly when setting a schedule on ZE50 car.
In the following example, I requested the schedule to be enabled. And the request sets "activated" to false:

`2021-01-26 10:03:58 DEBUG (MainThread) [renault_api.kamereon] Received Kamereon response 200 on GET https://api-wired-prod-1-euw1.wrd-aws.com/commerce/v1/accounts/8244b99e-5ec1-4eb7-8695-2e0de0a7d427/kamereon/kca/car-adapter/v1/cars/VFXXXXXXXX/charging-settings?country=FR: {"data":{"type":"Car","id":"VFXXXXXXXX","attributes":{"mode":"scheduled","schedules":[{"id":1,"activated":true,"monday":{"startTime":"T00:00Z","duration":480},"tuesday":{"startTime":"T00:00Z","duration":480},"wednesday":{"startTime":"T00:00Z","duration":480},"thursday":{"startTime":"T00:00Z","duration":480},"friday":{"startTime":"T00:00Z","duration":480},"saturday":{"startTime":"T00:00Z","duration":480},"sunday":{"startTime":"T00:00Z","duration":480}},{"id":2,"activated":false,"monday":{"startTime":"T00:00Z","duration":480},"tuesday":{"startTime":"T00:00Z","duration":480},"wednesday":{"startTime":"T00:00Z","duration":480},"thursday":{"startTime":"T00:00Z","duration":480},"friday":{"startTime":"T00:00Z","duration":480},"saturday":{"startTime":"T00:00Z","duration":480},"sunday":{"startTime":"T00:00Z","duration":480}},{"id":3,"activated":false},{"id":4,"activated":false},{"id":5,"activated":false}]}}} (with body data None) 2021-01-26 10:03:58 DEBUG (MainThread) [custom_components.renault.services] Charge set schedules attempt: {'id': 2, 'activated': True, 'monday': {'startTime': 'T23:30Z', 'duration': 30}, 'tuesday': {'startTime': 'T23:30Z', 'duration': 30}, 'wednesday': {'startTime': 'T23:30Z', 'duration': 30}, 'thursday': {'startTime': 'T23:30Z', 'duration': 30}, 'friday': {'startTime': 'T23:30Z', 'duration': 30}, 'saturday': {'startTime': 'T23:30Z', 'duration': 30}, 'sunday': {'startTime': 'T23:30Z', 'duration': 30}} 2021-01-26 10:03:59 DEBUG (MainThread) [renault_api.kamereon] Received Kamereon response 200 on POST https://api-wired-prod-1-euw1.wrd-aws.com/commerce/v1/accounts/8244b99e-5ec1-4eb7-8695-2e0de0a7d427/kamereon/kca/car-adapter/v2/cars/VFXXXXXXXX/actions/charge-schedule?country=FR: {"data":{"type":"ChargeSchedule","id":"ba40940e-7f9b-4761-b001-db84e0682330","attributes":{"schedules":[{"id":1,"activated":true,"monday":{"startTime":"T00:00Z","duration":480},"tuesday":{"startTime":"T00:00Z","duration":480},"wednesday":{"startTime":"T00:00Z","duration":480},"thursday":{"startTime":"T00:00Z","duration":480},"friday":{"startTime":"T00:00Z","duration":480},"saturday":{"startTime":"T00:00Z","duration":480},"sunday":{"startTime":"T00:00Z","duration":480}},{"id":2,"activated":false,"monday":{"startTime":"T23:30Z","duration":30},"tuesday":{"startTime":"T23:30Z","duration":30},"wednesday":{"startTime":"T23:30Z","duration":30},"thursday":{"startTime":"T23:30Z","duration":30},"friday":{"startTime":"T23:30Z","duration":30},"saturday":{"startTime":"T23:30Z","duration":30},"sunday":{"startTime":"T23:30Z","duration":30}},{"id":3,"activated":false},{"id":4,"activated":false},{"id":5,"activated":false}]}}} (with body data {'data': {'type': 'ChargeSchedule', 'attributes': {'schedules': [{'id': 1, 'activated': True, 'monday': {'startTime': 'T00:00Z', 'duration': 480}, 'tuesday': {'startTime': 'T00:00Z', 'duration': 480}, 'wednesday': {'startTime': 'T00:00Z', 'duration': 480}, 'thursday': {'startTime': 'T00:00Z', 'duration': 480}, 'friday': {'startTime': 'T00:00Z', 'duration': 480}, 'saturday': {'startTime': 'T00:00Z', 'duration': 480}, 'sunday': {'startTime': 'T00:00Z', 'duration': 480}}, {'id': 2, 'activated': False, 'monday': {'startTime': 'T23:30Z', 'duration': 30}, 'tuesday': {'startTime': 'T23:30Z', 'duration': 30}, 'wednesday': {'startTime': 'T23:30Z', 'duration': 30}, 'thursday': {'startTime': 'T23:30Z', 'duration': 30}, 'friday': {'startTime': 'T23:30Z', 'duration': 30}, 'saturday': {'startTime': 'T23:30Z', 'duration': 30}, 'sunday': {'startTime': 'T23:30Z', 'duration': 30}}, {'id': 3, 'activated': False, 'monday': None, 'tuesday': None, 'wednesday': None, 'thursday': None, 'friday': None, 'saturday': None, 'sunday': None}, {'id': 4, 'activated': False, 'monday': None, 'tuesday': None, 'wednesday': None, 'thursday': None, 'friday': None, 'saturday': None, 'sunday': None}, {'id': 5, 'activated': False, 'monday': None, 'tuesday': None, 'wednesday': None, 'thursday': None, 'friday': None, 'saturday': None, 'sunday': None}]}}}) 2021-01-26 10:03:59 INFO (MainThread) [custom_components.renault.services] Charge set schedules result: KamereonVehicleChargeScheduleActionData(raw_data={'schedules': [{'id': 1, 'activated': True, 'monday': {'startTime': 'T00:00Z', 'duration': 480}, 'tuesday': {'startTime': 'T00:00Z', 'duration': 480}, 'wednesday': {'startTime': 'T00:00Z', 'duration': 480}, 'thursday': {'startTime': 'T00:00Z', 'duration': 480}, 'friday': {'startTime': 'T00:00Z', 'duration': 480}, 'saturday': {'startTime': 'T00:00Z', 'duration': 480}, 'sunday': {'startTime': 'T00:00Z', 'duration': 480}}, {'id': 2, 'activated': False, 'monday': {'startTime': 'T23:30Z', 'duration': 30}, 'tuesday': {'startTime': 'T23:30Z', 'duration': 30}, 'wednesday': {'startTime': 'T23:30Z', 'duration': 30}, 'thursday': {'startTime': 'T23:30Z', 'duration': 30}, 'friday': {'startTime': 'T23:30Z', 'duration': 30}, 'saturday': {'startTime': 'T23:30Z', 'duration': 30}, 'sunday': {'startTime': 'T23:30Z', 'duration': 30}}, {'id': 3, 'activated': False}, {'id': 4, 'activated': False}, {'id': 5, 'activated': False}]}) 2021-01-26 10:03:59 INFO (MainThread) [custom_components.renault.services] It may take some time before these changes are reflected in your vehicle.`

The same happens if I try to disable a schedule. The change isn't reflected.

**Comment by epenet** [comment]
After investigation, this needs to be first implemented in [renault-api](https://github.com/hacf-fr/renault-api): [src/renault_api/kamereon/helpers.py](https://github.com/hacf-fr/renault-api/blob/main/src/renault_api/kamereon/helpers.py)

Once `update_schedule` is updated there, and released, then it should be sufficient to bump just the version number in `hassRenaultZE`.

**Comment by epenet** [comment]
@eviollet now that `renault-api` v0.1.0 is released, I have pushed a commit to the master branch which should fix this.

Can you try it?

**Comment by epenet** [comment]
@eviollet can we close this issue now?

**Comment by eviollet** [comment]
> @eviollet can we close this issue now?

Yes, I'm sorry, I forgot to answer the issue. Everything is working as intended now.

Thank you very much.
